### DISCUSSION TITLE
Authentication Message on login

### DISCUSSION THREAD

**Discussion by angelogl** [serenity-is/Serenity#7199] [General]
Hello everyone, I hope you are ok. How can change the message "Invalid username or password" when the user has the field "IsActive" equal false. If the user is inactive in the user form I would like to get a message like "User is Inactive". And, if I add a ExpiredDate to the user form, when I validate it get the message "User with a expired date". Thank a lot, happy day.

**Comment by VictorTomaili** [comment]
I suggest not showing messages regarding inactive and expired accounts. It allows anyone to determine if this email or username is used in the system or not. For security reasons, you can send an email instead.
